src/client/notifications.ts

```diff
@@ -0,0 +1,17 @@
+const icon = '/images/logo-transparent.svg';
+
+const sendNotification = (options, route) => {
```

Comment:
To define arguments typings:

```suggestion
const sendNotification = (options: NotificationOptions, route: () => void) => {
```